cypress/e2e/WebInterface/Test Cases/TestCaseImport/TestCaseImportviaFile.cy.ts

```diff
@@ -0,0 +1,338 @@
+                        headers: {
+                            authorization: 'Bearer ' + accessToken.value
+                        },
+                        method: 'GET',
+                    }).then((response) => {
+                        expect(response.status).to.eql(200)
+                        expect(response.body.json).to.be.exist
+                        switch (i) {
+                            case 0:
+                                expect(response.body.json).to.eql(testCaseJsonTstSmallBatch00)
+                                break;
+                            case 1:
+                                expect(response.body.json).to.eql(testCaseJsonTstSmallBatch01)
+                                break;
+                            case 2:
+                                expect(response.body.json).to.eql(testCaseJsonTstSmallBatch02)
+                                break;
+                            case 3:
+                                expect(response.body.json).to.eql(testCaseJsonTstSmallBatch03)
+                                break;
+                            case 4:
+                                expect(response.body.json).to.eql(testCaseJsonTstSmallBatch04)
+                                break;
+                            case 5:
+                                expect(response.body.json).to.eql(testCaseJsonTstSmallBatch05)
+                                break;
+                        }
+                    })
+                })
+            })
+            OktaLogin.Login()
+            //click on created measure
+            MeasuresPage.clickEditforCreatedMeasure()
+            cy.get(EditMeasurePage.testCasesTab).click()
+        }
+
+    })
+
+    it('Import new test case onto an existing measure -- large batch file', () => {
+        //navigate to the main measures page
+        cy.get(Header.measures).click()
+
+        //click on created measure
+        MeasuresPage.clickEditforCreatedMeasure()
+
+        //click on the test case tab
+        cy.get(EditMeasurePage.testCasesTab).click()
+
+        //click on the import test case button
+        cy.get(TestCasesPage.importTestCasesBtn).click()
+
+        //select file
+        cy.get(TestCasesPage.filAttachDropBox).attachFile(largeBatchTestCaseFile)
+
+        //import moadal should contain test case name
+        cy.get(TestCasesPage.importTestCaseModalList).should('contain.text', 'EncounterStatusMultiPerson (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1)')
```

Comment:
what is all of these (1)s?

Reply:
I should have probably went back and made smaller names for the larger batch file that I created 125 tests -- those had the ever increasing number of "(1)"s because I simply cloned the first test case over and over again to get 125, in that one measure. Then, I just left the names the same, as was created by doing so.  This made the names really long ....so, in a way, by doing that, I also verified that long names on import won't cause any issues,